mapreduce/src/main/java/com/marklogic/mapreduce/MarkLogicOutputFormat.java

```diff
@@ -99,6 +99,7 @@ public void checkOutputSpecs(JobContext context) throws IOException,
                 if (ex.getCause() instanceof ServerConnectionException) {
                     LOG.warn("Unable to connect to " + hosts[i]
                             + " to query destination information");
+                    LOG.debug(ex);
```

Comment:
The practice in mlcp and hadoop connector is we always check `if (LOG.isDebugEnabled())` before we log out as DEBUG level.